src/cityreader/cityreader.py

```diff
@@ -28,6 +30,12 @@ def cityreader(cities=[]):
     # TODO Implement the functionality to read from the 'cities.csv' file
     # For each city record, create a new City instance and add it to the
     # `cities` list
+    with open('cities.csv', 'r') as csv_file:
+        csv_reader = csv.reader(csv_file)
+        next(csv_reader)
+        for city in csv_reader:
+            print(f'\n{city}')
+            cities.append(City(city[0], city[3], city[4]))
```

Comment:
Look at the test files again and think about what it's expecting to see. The lines from the csv are read in as strings but they aren't necessarily strings as the datatype. You're close but you just need to change 2 things here